### ISSUE TITLE
Autocomplete for Find Type

### ISSUE THREAD

**Issue by mehmetf** [dart-atom/dartlang#1068]
It would be great if find type in the atom plugin for Dart supported autocomplete. Right now, I have to type a search term and press Enter which will show me results in the right window pane:

![image](https://cloud.githubusercontent.com/assets/6687929/16956387/2512d6de-4d8d-11e6-8d2f-4a352694da2c.png)

Find file on the other hand has autocomplete and is all kinds of wonderful:

![image](https://cloud.githubusercontent.com/assets/6687929/16956424/3e42826c-4d8d-11e6-8487-64e9b7d62bd7.png)

**Comment by devoncarew** [comment]
Makes sense - we should try and hook into whatever list filtering they're doing.
